src/popup/router/components/AccountInfo.vue

```diff
@@ -61,22 +61,18 @@ export default {
   },
   mixins: [CopyMixin],
   props: {
-    accountIdx: { type: Number, default: -1 },
     color: { type: String, default: 'black' },
-    showCopyIcon: { type: Boolean, default: false },
+    canCopyAddress: Boolean,
+    i: { type: Number, default: 0 },
```

Comment:
I strongly suggest to not use one letter variables/prop names. Hard to find, refactor and know the context.